packages/@lightningjs/ui-components/src/components/Knob/Knob.d.ts

```diff
@@ -18,17 +18,19 @@
 
 import lng from '@lightningjs/core';
 import Base from '../Base';
-import type { Color, StylePartial } from '../../types/lui';
+import { Color, StylePartial } from '../../types/lui';
 
-export type KnobStyle = {
+type KnobStyle = {
   circleColor: Color;
   radius: lng.Tools.CornerRadius;
 };
 
```

Comment:
I think you still need to add the namespace and then update the class with the namespace but should check with Anthony
```
declare namespace Knob {
  export type TemplateSpec = Base.TemplateSpec;
}
```

Reply:
nah in cases where we aren't making changes to the TemplateSpec or TypeConfig we should omit them and just pass the parent's interfaces